package-lock.json

```diff
@@ -1481,6 +1481,21 @@
       "integrity": "sha512-RbzJvlNzmRq5c3O09UipeuXno4tA1FE6ikOjxZK0tuxVv3412l64l5t1W5pj4+rJq9vpkm/kwiR07aZXnsKPxw==",
       "dev": true
     },
+    "@tsconfig/node14": {
```

Comment:
Should we regenerate the package-lock with npm 8?

Reply:
I think if we're going to ride out node@v14 until end of LTS, we should stick with npm 6. But I'm open to targeting node@v16 now...